src/web/services/user/profile.rs

```diff
@@ -143,7 +144,44 @@ async fn profile(
     }
 
     // Render the profile template and send to user.
-    return template.in_page(&req, page_title).await;
+    let mut page = template.in_page(&req, page_title.clone()).await?;
+
+    let mut tags = Tags::default();
+    tags.title = page_title.clone();
+    tags.url = format!("/user/{}", id);
+    let mut description = String::new();
+    description.push_str(target_user.role.to_string().as_str());
+    description.push_str("\n");
```

Comment:
This could all be replaced with a `let mut description = format!("{}\n", target_user.role);` This is preferable from a simplicity standpoint.